lib/Option/SanitizerOptions.cpp

```diff
@@ -37,12 +37,11 @@ struct Info {
   const char *file;
 };
 
-constexpr std::array<Info, 4> infos = {{
-    {SanitizerKind::Address,    "address",    "asan"},
-    {SanitizerKind::Thread,     "thread",     "tsan"},
-    {SanitizerKind::Undefined,  "undefined",  "ubsan"},
-    {SanitizerKind::Fuzzer,     "fuzzer",     "fuzzer"},
-}};
+#define SANITIZER(kind, name, file) Info{SanitizerKind::kind, #name, #file},
+constexpr std::array<Info, 4> infos = {
+  #include "Sanitizer.def"
```

Comment:
Oh, I meant that everything that's using this "infos" could operate directly on the .def file. I don't know if you'd consider that better or worse, though.

Reply:
Ahh, got it.

Reply:
I think that it might be easier to read the way that @jrose-apple suggests, at the cost of a slightly larger binary.